### DISCUSSION TITLE
Machinaris docker on Windows plotter doesn't start plotting

### DISCUSSION THREAD

**Discussion by myevit** [guydavis/machinaris#472] [Q&A] [answered]
Docker Desktop 4.3.0 (71786)

machinaris tested v0.6.1,v0.6.5, v0.6.6,v0.6.6, 

Full node running v0.6.1 reporting that plotting on remore plotter is running but doesn't show any plots in progress

[docker.log](https://github.com/guydavis/machinaris/files/7654638/docker.log)

version: '3.7'
services:

    machinaris:
        image: ghcr.io/guydavis/machinaris
        container_name: machinaris
        hostname: myePC
        restart: always
        volumes:
            - ~/.machinaris:/root/.chia
            - "E:/plots:/plots1"
            - "D:/plotting:/plotting"
        environment:
            - TZ=America/Edmonton
            - farmer_address=192.168.1.4
            - farmer_port=8447
            - farmer_pk=xxxxxxxxxxxx
            - pool_pk=xxxxxxxxxxxxxxx
            - pool_contract_address=xxxxxxxxxxxxx
            - mode=harvester,plotter
            - controller_host=192.168.1.4
            - controller_api_port=8927
            - worker_address=192.168.1.214
            - worker_api_port=8927
            - plots_dir=/plots1
            - blockchains=chia
        ports:
            - 8926:8926
            - 8927:8927

**Comment by guydavis** [comment]
Hi, please use the latest release.  v0.6.7.  Then follow the [troubleshooting](https://github.com/guydavis/machinaris/wiki/Troubleshooting) steps on wiki.

**Comment by myevit** [comment, marked as answer]
Interestly enough. Installed ubuntu on WSL2 with shared windows-ubuntu docker. started docker comporse from there (with changed mapped folders to /mnt/e/plots and /mnt/d/plots) and it started plotting with no problem. Something to do with folder mapping???

**Reply by guydavis** [reply]
Chances are you attempted to start plotting too soon after starting the container.  Please ensure you are running v0.6.7 on every Machinaris system.  There is better error message when launching Plotting in v0.6.7.  Glad you were able to get it working.  Please mark this discussion as Answered.
